packages/db-objects-jsonl/lib/objects/objectsInMemJsonlDB.js

```diff
@@ -43,6 +43,14 @@ class ObjectsInMemoryJsonlDB extends ObjectsInMemoryFileDB {
             throttleFS: {
                 intervalMs: 60000,
                 maxBufferedCommands: 1000
+            },
+            lockfile: {
+                // 5 retries starting at 250ms add up to just above 2s,
+                // so the DB has 3 more seconds to load all data if it wants to stay within the 5s connectionTimeout
+                retries: 5,
+                retryMinTimeoutMs: 250,
+                // This makes sure the DB stays locked for maximum 2s even if the process crashes
+                staleMs: 2000
```

Comment:
We should still check for installer if we can give a restart dalay for the systemd service